.github/workflows/promote-beta-experimental-opt-in.yml

```diff
@@ -0,0 +1,37 @@
+# Promotes the current Nightly channel to Beta/Experimental opt-in on a schedule.
+
+name: Promote Beta/Experimental opt-in Channels
+
+on:
```

Comment:
We will need a way to trigger manually as well, can be done with the workflow_dispatch event - https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#workflow_dispatch

Do we need to allow an amp version override?

Reply:
I'm not sure if that's needed. We can (and should) make any manual changes with a separate job or directly in the versioning.json file. wdyt?